src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
Tiles aren't meters most of the time. Compare them to other relevant values: shrieker's shriek has 50 volume and its a zombie specifically mutated to shout loudly.